src/module_3/train.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import pandas as pd
+import pickle
+
+from pathlib import Path
+from sklearn.base import BaseEstimator
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import precision_recall_curve, auc, roc_auc_score
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from typing import Tuple
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+C_TUNING = [1e-8, 1e-6, 1e-4, 1e-2, 1]
+IMPORTANT_COLS = ["ordered_before", "abandoned_before", "global_popularity"]
+
+TARGET_COL = "outcome"
+
+HOLDOUT = 0.8
+
+DATA_PATH = "/home/alejandro/Zrive"
+
+
+def evaluate_model(model_name: str, y_test: pd.Series, y_pred: pd.Series) -> float:
+    """
+    Evaluate the model using precision-recall AUC and ROC AUC
+
+    Args:
+        model_name (str): The name of the model being evaluated.
+        y_test (pd.Series): The true labels of the test set.
+        y_pred (pd.Series): The predicted labels of the test set.
+
+    Returns:
+        float: The precision-recall AUC score of the model.
+    """
+    precision_, recall_, _ = precision_recall_curve(y_test, y_pred)
+    pr_auc = auc(recall_, precision_)
+    roc_auc = roc_auc_score(y_test, y_pred)
+    logging.info(f"{model_name} - PR AUC: {pr_auc:.2f}, ROC AUC: {roc_auc:.2f}")
+    return pr_auc
+
+
+def feature_split(df: pd.DataFrame) -> Tuple[pd.DataFrame, pd.Series]:
```

Comment:
Good that you have abstractred this into a method. However, it makes more sense to have it a utils.py since inference will also use ir